client/src/modules/creditors/groups/creditor-groups-list.html

```diff
@@ -27,4 +54,9 @@ <h5 style="margin-top : 0px;"></span><span translate> FORM.LABELS.ACCOUNT_NUMBER
 
     </div>
   </div>
+
+  <div class="row">
+    <br>
+    <p style="color:red" translate>CREDITOR_GROUP.NO_CREDITOR_GROU_ENRED_YET</p>
```

Comment:
In order to keep the application looking clean and uniform, use bootstrap classes for coloring.  You can find the [whole list in the documentation](https://getbootstrap.com/docs/3.3/css/#helper-classes-colors).